packages/transition-frontend/src/components/forms/preferences/sections/PreferencesSectionGeneral.tsx

```diff
@@ -14,25 +15,28 @@ import { roundToDecimals } from 'chaire-lib-common/lib/utils/MathUtils';
 import InputStringFormatted from 'chaire-lib-frontend/lib/components/input/InputStringFormatted';
 import InputWrapper from 'chaire-lib-frontend/lib/components/input/InputWrapper';
 import InputSelect from 'chaire-lib-frontend/lib/components/input/InputSelect';
+import config from 'chaire-lib-common/lib/config/shared/project.config';
```

Comment:
At least, can we have a //TODO with each of these new imports to point to the task about removing them

Reply:
This will never be removed, as we still need the list of enabled sections to list them for the user to select the default section. Same applies to the other comment

Reply:
This should be served by an API to the frontend then, not as an hardcoded list for that purpose ?

Reply:
Sure, it's an improvement, but I will not miss a chance ask for it. It might end up being done at some point... :)

Reply:
And yes conditional webpack compile might be a good way to do. I just don't like have just a generic import and having every bit of the config easily available... We never know what we might leak!

Reply:
> Sure, it's an improvement, but I will not miss a chance ask for it. It might end up being done at some point... :)

There's an issue for it now fwiw!